pennylane/ops/qubit/state_preparation.py

```diff
@@ -232,6 +232,11 @@ def state_vector(self, wire_order=None):
         return math.convert_like(ket, op_vector)
 
 
+# pylint: disable=missing-class-docstring
+class QubitStateVector(StatePrep):
+    pass  # QSV is still available
```

Comment:
Do we want to keep `name` to be `"QubitStateVector"` still? Otherwise, we will still need to update all the plugins.

Reply:
It is `QubitStateVector` by default because it inherits the `__init__` method of `Operator` which sets `self._name = self.__class__.__name__`.

Reply:
oh right sorry.